avocado/core/varianter.py

```diff
@@ -45,9 +45,29 @@ def generate_variant_id(variant):
     :return: String compounded of ordered node names and a hash of all
              values.
     """
+    def variantname(variant):
```

Comment:
Maybe this should be part of the `Varianter()` class, so we can call like this:

```python3
varianter.name
```
or

```python3
varianter.get_name()
```

Reply:
@beraldoleal 
I guess it is not a good idea, `generate_variant_id` is a independent method and does not have reference to `Varianter:` class , more over not seeing a usecase currently for `varianter.name` outside of `generate_variant_id`.

I feel it is good to contain it in this function itself for now.

Reply:
I'm not familiar with this code, but you are passing a `variant` argument, right? Is this a `Varianter()` object? If yes, then IMO this should be part of Variant() class, if not, then ignore my comment.

Reply:
sure, np, `variant` is a list of `MuxTreeNode` object.